assets/js/hooks/useGlobalTrackingEffect.test.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * `useGlobalTrackingEffect` hook tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	provideSiteInfo,
+	renderHook,
+	waitForDefaultTimeouts,
+} from '../../../tests/js/test-utils';
+import { useGlobalTrackingEffect } from './useGlobalTrackingEffect';
+import * as tracking from '../util/tracking';
+import { setItem, setStorageOrder } from '../googlesitekit/api/cache';
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../googlesitekit/constants';
+
+const mockTrackEvent = jest
+	.spyOn( tracking, 'trackEvent' )
+	.mockImplementation( () => Promise.resolve() );
+
+describe( 'useGlobalTrackingEffect', () => {
+	let registry;
+
+	beforeAll( () => {
+		setStorageOrder( [ 'localStorage', 'sessionStorage' ] );
+	} );
```

Comment:
Why are we setting this to prefer `localStorage`? I don't think it should be necessary to set/change.

Reply:
Since by default order it is setting `sessionStorage` for the `storageBackend` backend. And we are running this in test environment it made more sense to me to use  `localStorage` for stability, even our storage mechanism prefers to use `sessionStorage` for some reason

Reply:
Both are mocked so one shouldn't be more stable than another. It sounds like it was failing from checking the wrong storage but we shouldn't even need to be specific about which one and instead use our `getStorage` utility.